app/assets/stylesheets/application.scss

```diff
@@ -2,3 +2,5 @@
 @import "bootstrap";
 
 @import "layout";
+@import "table/head";
+@import "table/body";
```

Comment:
Regarding these two files: I decided to split them into two files ("_table_head.scss" and "_table_body.scss"), and then decided to move them into a new folder ("table"). Is this ok? Should they be a single file instead?